mailpoet/lib/Newsletter/NewslettersRepository.php

```diff
@@ -201,6 +203,8 @@ public function getAnalytics(): array {
       'product_purchased_in_category_emails_count' => $analyticsMap[NewsletterEntity::TYPE_AUTOMATIC][PurchasedInCategory::SLUG] ?? 0,
       'abandoned_cart_emails_count' => $analyticsMap[NewsletterEntity::TYPE_AUTOMATIC][AbandonedCart::SLUG] ?? 0,
     ];
+    $data['campaigns_count'] = $data['welcome_newsletters_count'] + $data['notifications_count'] + $data['automatic_emails_count'] + $data['automation_emails_count'] + $data['re-engagement_emails_count'] + $data['sent_newsletters_count'];
```

Comment:
Should we just use `array_sum($analyticsMap)`, maybe? That would include any possible new future email types. What do you consider better or safer?